src/features/ballot/components/AllocationList.tsx

```diff
@@ -79,91 +54,50 @@ export function AllocationFormWrapper({
   });
 
   return (
-    <AllocationListWrapper>
-      <Table>
-        {renderHeader?.()}
-        <Tbody>
-          {fields.map((project, i) => {
-            const idx = i;
+    <Table>
+      {renderHeader?.()}
+      <Tbody>
+        {fields.map((project, i) => {
+          const idx = i;
```

Comment:
why not call it directly idx on line 60?

Reply:
good question, I just don't modify it, let me modify it